config/settings.py

```diff
@@ -28,7 +29,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config('DEBUG', default=False, cast=bool)
 
-ALLOWED_HOSTS = ["*"]
+ALLOWED_HOSTS = ['wonder-watch-live.herokuapp.com', 'localhost', '127.0.0.1']
```

Comment:
It's better to make this an environment variable:

.env
ALLOWED_HOSTS=.localhost, .herokuapp.com

settings.py
from decouple import config, Csv

ALLOWED_HOSTS = config('ALLOWED_HOSTS', cast=Csv())